Panic on debugging yandex cloud provider

I try to run yandex cloud plugin in debug mode with `config.hcl`:
```
cloudquery {
  plugin_directory = "./cq/providers"
  policy_directory = "./cq/policies"

  provider "yandex-cloud/yandex" {
    source  = "yandex-cloud"
    version = "latest"
  }

  connection {
    dsn = "host=localhost user=postgres password=pass database=postgres port=5432"
  }
}

// All Provider Configurations

provider "yandex" {
  configuration {
    // Optional. Filter as described https://cloud.yandex.com/docs/resource-manager/grpc/folder_service#List
    // folder_filter = ""
    // Optional. If not specified either using all folders accessible.
    // folder_ids = [<CHANGE_THIS_TO_YOUR_FOLDER_ID>]
    cloud_id = "<my_cloud_id>"
  }
  resources = [
    "compute.disks"
  ]
}
```

it panics with:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1b3a1ad]

goroutine 101 [running]:
github.com/cloudquery/cloudquery/pkg/ui/console.buildFetchProgress.func2(0xc000114a90, 0x6, 0x1d96b7a, 0x9, 0xc00045b560, 0xb3, 0x0, 0x0)
        /home/runner/work/cloudquery/cloudquery/pkg/ui/console/client.go:163 +0xad
github.com/cloudquery/cloudquery/pkg/client.(*Client).Fetch.func1(0x0, 0x0)
        /home/runner/work/cloudquery/cloudquery/pkg/client/client.go:257 +0x1069
golang.org/x/sync/errgroup.(*Group).Go.func1(0xc0002a7110, 0xc0000cc930)
        /home/runner/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:57 +0x59
created by golang.org/x/sync/errgroup.(*Group).Go
        /home/runner/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:54 +0x66
```

--- a/pkg/plugin/plugin.go
+++ b/pkg/plugin/plugin.go
@@ -125,7 +125,7 @@ func newUnmanagedPlugin(providerName string, config *plugin.ReattachConfig) (*un
 		return nil, fmt.Errorf("failed to cast plugin")
 	}
 	return &unmanagedPlugin{
-		name:     providerName,
+		name:     fmt.Sprintf("%s_%s", providerName, providerName),
 		config:   config,
 		client:   client,
 		provider: provider,

--- a/pkg/ui/console/client.go
+++ b/pkg/ui/console/client.go
@@ -237,6 +237,11 @@ func buildFetchProgress(ctx context.Context, providers []*config.Provider) (*Pro
 			return
 		}
 		bar := fetchProgress.GetBar(update.Provider)
+		if bar == nil {
+			fetchProgress.AbortAll()
+			ui.ColorizedOutput(ui.ColorError, "❌ console UI failure, fetch will complete shortly\n")
+			return
+		}
 		bar.b.IncrBy(update.DoneCount() - int(bar.b.Current()))
 
 		if bar.Status == ui.StatusError {

--- a/pkg/ui/console/progress.go
+++ b/pkg/ui/console/progress.go
@@ -156,6 +156,15 @@ func (u *Progress) GetBar(name string) *Bar {
 	return bar
 }
 
+func (u *Progress) AbortAll() {
+	u.lock.RLock()
+	defer u.lock.RUnlock()
+	for _, b := range u.bars {
+		b.b.Abort(true)
+	}
+	u.Wait()
+}
+
 func defaultStatusUpdater(u *Bar, _ decor.Statistics) string {
 	return emojiStatus[u.Status]
 }

--- a/pkg/ui/console/progress.go
+++ b/pkg/ui/console/progress.go
@@ -35,7 +35,7 @@ func (b *Bar) SetTotal(total int64, triggerComplete bool) {
 type Progress struct {
 	p       *mpb.Progress
 	bars    map[string]*Bar
-	lock    sync.RWMutex
+	lock    sync.Mutex
 	options *ProgressOptions
 }
 
@@ -76,8 +76,8 @@ func (u *Progress) Add(name, displayName, message string, total int64) {
 					if u.options.StatusFunc == nil {
 						return ""
 					}
-					u.lock.RLock()
-					defer u.lock.RUnlock()
+					u.lock.Lock()
+					defer u.lock.Unlock()
 					uiBar := u.bars[name]
 					return u.options.StatusFunc(uiBar, statistics)
 				}, decor.WC{W: 2, C: 1}),
@@ -87,8 +87,8 @@ func (u *Progress) Add(name, displayName, message string, total int64) {
 				if u.options.MessageHook == nil {
 					return ""
 				}
-				u.lock.RLock()
-				defer u.lock.RUnlock()
+				u.lock.Lock()
+				defer u.lock.Unlock()
 				uiBar := u.bars[name]
 				return u.options.MessageHook(uiBar, statistics)
 			}, decor.WC{W: len(name) + 1, C: decor.DidentRight}),
@@ -107,8 +107,8 @@ func (u *Progress) Add(name, displayName, message string, total int64) {
 }
 
 func (u *Progress) Increment(name string, n int) {
-	u.lock.RLock()
-	defer u.lock.RUnlock()
+	u.lock.Lock()
+	defer u.lock.Lock()
 	bar, ok := u.bars[name]
 	if !ok {
 		return
@@ -117,8 +117,8 @@ func (u *Progress) Increment(name string, n int) {
 }
 
 func (u *Progress) Update(name, status, msg string, n int) {
-	u.lock.RLock()
-	defer u.lock.RUnlock()
+	u.lock.Lock()
+	defer u.lock.Unlock()
 	bar, ok := u.bars[name]
 	if !ok {
 		return
@@ -131,8 +131,8 @@ func (u *Progress) Update(name, status, msg string, n int) {
 }
 
 func (u *Progress) AttachReader(name string, data io.Reader) io.Reader {
-	u.lock.RLock()
-	defer u.lock.RUnlock()
+	u.lock.Lock()
+	defer u.lock.Unlock()
 	bar, ok := u.bars[name]
 	if !ok {
 		return data
@@ -147,8 +147,8 @@ func (u *Progress) Wait() {
 }
 
 func (u *Progress) GetBar(name string) *Bar {
-	u.lock.RLock()
-	defer u.lock.RUnlock()
+	u.lock.Lock()
+	defer u.lock.Unlock()
 	bar, ok := u.bars[name]
 	if !ok {
 		return nil
@@ -157,8 +157,8 @@ func (u *Progress) GetBar(name string) *Bar {
 }
 
 func (u *Progress) AbortAll() {
-	u.lock.RLock()
-	defer u.lock.RUnlock()
+	u.lock.Lock()
+	defer u.lock.Unlock()
 	for _, b := range u.bars {
 		b.b.Abort(true)
 	}

